maven.gradle

```diff
@@ -1,12 +1,27 @@
-import com.infinum.maven.SonatypeConfiguration
-
-SonatypeConfiguration.metaClass.constructor = { ->
-    def constructor = SonatypeConfiguration.class.getConstructor()
-    def instance = constructor.newInstance()
-    instance.load()
-    instance
+def properties = new Properties()
+def propsFile = file("$rootDir/publish.properties")
+if (propsFile.exists()) {
+    properties.load(propsFile.newInputStream())
 }
 
-ext {
-    sonatype = new SonatypeConfiguration()
-}
\ No newline at end of file
+/*
+* 1. findProperty("sonatypeUsername"): use parameter from the command (if attached) so you can run ->
+* /gradlew deploy -PsonatypeUsername={yourMavenUsername} -PsonatypePassword={yourMavenPassword}.
+* 2. properties["sonatype.user"]: use property defined in publish.properties file in the root directory. If you have that file in the root
+* of the project you can run -> ./gradlew deploy
+* 3. System.getenv("SONATYPE_PASS"): use global variable defined in the system. No need to create publish.properties file, you can simply
```

Comment:
Maybe it would be better to generically explain what are the ways in which these can be provided, this is a bit inconsistent, here you mention the "pass" property but before you were talking about "user".

Reply:
I can do that 👍

Reply:
Aha, these were only examples but considering it wasn't clear I should update it 👍